dashboard/config/routes.rb

```diff
@@ -12,12 +12,14 @@ def redirect_to_teacher_dashboard
 
   resources :user_levels, only: [:update]
 
-  resources :gallery_activities, path: '/gallery' do
+  resources :gallery_activities, only: [:index], path: '/gallery' do
```

Comment:
do we need to move the `gallery_activities#destroy` action somewhere else as well, or is it currently unused?